src/file_analysis/analyzer/extract/Extract.cc

```diff
@@ -34,7 +34,7 @@ Extract::~Extract()
 		util::safe_close(fd);
 	}
 
-static const ValPtr& get_extract_field_val(const RecordValPtr& args,
+static ValPtr get_extract_field_val(const RecordValPtr& args,
                                            const char* name)
```

Comment:
Indentation

Reply:
Fix manually, or wait on subsequent pretty-printing?

Reply:
From my point of view - until we have a pretty-printer, we should fix these manually :)

Reply:
For that example, what would be the right fix?  Use only spaces, since there's no active indentation?